bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/handler/PLCDigitalHandler.java

```diff
@@ -266,13 +265,13 @@ protected void doInitialization() {
             tBuilder.withLabel(label);
 
             for (int i = 0; i < getNumberOfChannels(); i++) {
-                String name = kind + String.valueOf(i + 1);
+                String name = kind + i + 1;
```

Comment:
Sorry about the noise, but from my point of view, this will break the binding. For `i = 0`, the name will not be `I1`, but `I01`. Miss i something?

Reply:
Yes, this is certainly wrong. Thanks!

Reply:
The line seems not be reverted

Reply:
Sorry for that. Now it seems there are no further occurrences of this in the PR....